app/src/main/java/io/rapidpro/androidchannel/RapidPro.java

```diff
@@ -85,7 +85,7 @@ public class RapidPro extends Application {
     public static long MESSAGE_THROTTLE_WINDOW = 1000 * 60 * (MESSAGE_THROTTLE_MINUTES + 2);
     public static final long MESSAGE_RATE_LIMITER = 1000;
 
-    public static final String MESSAGE_PACK_VERSION_SUPPORTED = "1.1";
+    public static final String MESSAGE_PACK_VERSION_SUPPORTED = "1.2";
```

Comment:
We need to change the packs version since we change

https://github.com/rapidpro/android-channel/pull/31/files#diff-70786d593e50ed4617657bf6cbe08e389091d25c2c7b0d6c143962c7c193ff5aR72

We'll publish new APKs for that